src/rdkafka_fetcher.c

```diff
@@ -819,8 +819,10 @@ int rd_kafka_broker_fetch_toppars(rd_kafka_broker_t *rkb, rd_ts_t now) {
 
 
         /* FetchRequest header */
-        /* ReplicaId */
-        rd_kafka_buf_write_i32(rkbuf, -1);
+        if (rd_kafka_buf_ApiVersion(rkbuf) >= 15)
```

Comment:
Can use `ApiVersion` instead of `rd_kafka_buf_ApiVersion(rkbuf)`

Reply:
@pranavrth I think it's not necessary as it's a define, not a function call

Reply:
The condition must be
```c
if (rd_kafka_buf_ApiVersion(rkbuf) <= 14)
```